org.uqbar.project.wollok.typeSystem/src/org/uqbar/project/wollok/typesystem/AbstractContainerWollokType.xtend

```diff
@@ -35,13 +35,9 @@ abstract class AbstractContainerWollokType extends BasicType implements Concrete
  	}
 
  	override lookupMethod(String selector, List<?> parameterTypes) {		
-  		val m = container.lookupMethod(selector, parameterTypes, true)		
-  		// TODO: por ahora solo checkea misma cantidad de parametros		
-  		// 		debería en realidad checkear tipos !  		
-  		if (m !== null && m.parameters.size == parameterTypes.size)		
```

Comment:
This validation was wrong, not taking varargs into account. 
@fdodino I removed it because the previous part is already delegating to the same code which is run by the interpreter, so I think it should be enough.